Changelog.md

```diff
@@ -12,11 +12,13 @@ Compiler Features:
  * SMTChecker: Support ``delete``.
  * SMTChecker: Inline external function calls to ``this``.
  * Inline Assembly: Revert change introduced in 0.5.7: The ``callvalue()`` instruction does not require ``payable`` anymore.
+ * Inline Assembly: Disallow the combination of msize and the Yul optimizer.
```

Comment:
This means, `msize` can only be used in yul code when using the yul optimizer? Not in `inline assembly` code that is compiled via yul?

Reply:
The problem is that if the optimizer is applied to an isolated piece of code like the ABI encoder, then it might change an msize that appears in a totally different part of the code. Until we have a version of the code generator that creates a complete block of yul code for the full source code, we have to disable it like that, I fear.